src/evolution/EvolutionMethods.cpp

```diff
@@ -15,12 +15,14 @@ FluxBdrCoefficientsCentered bdrCentCoeff{
 	{BdrCond::PEC               ,	{2.0, 0.0}},
 	{BdrCond::PMC               ,	{0.0, 2.0}},
 	{BdrCond::SMA               ,	{0.0, 0.0}},
+	{BdrCond::SurfaceCond       ,   {1.0, 1.0}},
```

Comment:
Surface conductivity would appear as an Surface Impedance Boundary Condition (SIBC). Are you planning to implement that?

Reply:
I do not think that it is a simple number. That kind of conditions probably imply interconnecting with a 1D domain and/or possibly additional degrees of freedom.

If interconnecting with a 1D domain, probably the best is to use the standard numerical fluxes that normal propagation between elements use.